# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp
# Commit: e25543fae7ee
# Full Hash: e25543fae7ee03ac57b9cfbfeb2418c9e2f490fc
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r 4230bdcf07ae -r e25543fae7ee layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp
--- a/layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp	Wed Jun 22 16:15:57 2022 +0000
+++ b/layout/tools/layout-debug/src/nsLayoutDebuggingTools.cpp	Wed Jun 22 16:16:59 2022 +0000
@@ -255,7 +255,7 @@
 nsLayoutDebuggingTools::DumpCounterManager() {
   NS_ENSURE_TRUE(mDocShell, NS_ERROR_NOT_INITIALIZED);
   if (PresShell* presShell = GetPresShell(mDocShell)) {
-    presShell->FrameConstructor()->CounterManager()->Dump();
+    presShell->FrameConstructor()->GetContainStyleScopeManager().DumpCounters();
   }
   return NS_OK;
 }